test/module/shared_model/builders/protobuf/transport_builder_test.cpp

```diff
@@ -97,6 +99,22 @@ class TransportBuilderTest : public ::testing::Test {
         .build();
   }
 
+  auto createEmptyBlock() {
```

Comment:
This is code duplication for methods above. Please separate as `createValid` as method with `builder` input which calls 
```
builder.createdTime(created_time)....build()
```
, and `createInvalid` with 
```
builder..createdTime(123)....build()
```
, and use it both for 
```
TestEmptyBlockBuilder()
``` 
and 
```
TestBlockBuilder().transactions(std::vector<Transaction>({createTransaction()}))
```